dpu-api/gen/device_service.pb.go

```diff
@@ -0,0 +1,314 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
Definately should commit this in a separate commit and need a `Makefile` rule

Reply:
added to my to-do